src/components/svg-wrapper.tsx

```diff
@@ -134,10 +139,38 @@ const SvgWrapper = () => {
                 dispatch(clearSelected());
             }
             // console.log(x, y, svgViewBoxMin);
+        } else if (mode === 'select') {
+            dispatch(
+                setSelectStart({
+                    x: (x * svgViewBoxZoom) / 100 + svgViewBoxMin.x,
+                    y: (y * svgViewBoxZoom) / 100 + svgViewBoxMin.y,
+                })
+            );
+            dispatch(
+                setSelectMoving({
+                    x: (x * svgViewBoxZoom) / 100 + svgViewBoxMin.x,
+                    y: (y * svgViewBoxZoom) / 100 + svgViewBoxMin.y,
+                })
+            );
         }
     });
     const handleBackgroundMove = useEvent((e: React.PointerEvent<SVGSVGElement>) => {
-        if (active === 'background') {
+        if (mode === 'select') {
+            if (selectStart.x != 0 && selectStart.y != 0) {
+                const { x, y } = getMousePosition(e);
+                // console.log(
+                //     'MV: ',
+                //     (x * svgViewBoxZoom) / 100 + svgViewBoxMin.x,
+                //     (y * svgViewBoxZoom) / 100 + svgViewBoxMin.y
+                // );
+                dispatch(
+                    setSelectMoving({
```

Comment:
This could also be removed right? And then `selectStart` and `selectMoving` won't be shared across files :)

Reply:
But, how can our selection-box follow users' mouse?

Reply:
Oh sorry, I commented in the wrong place. It should be `handlePointerMove` in `src\components\svg-canvas-graph.tsx`.